src/timer.h

```diff
@@ -1,49 +1,54 @@
 /*
  * This file is
  * Copyright (c) 2009 by Jim Fougeron jfoug AT cox dot net
- * Copyright (c) 2019 by magnum
+ * Copyright (c) 2019-2020 by magnum
  *
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted.
- *
- * Portable hi-res timer.  Was a nice C++ class. Downgraded to C for john.
  */
 
 #ifndef _HAVE_TIMER_H
 #define _HAVE_TIMER_H
```

Comment:
I think we should use `_JOHN` in place of `_HAVE` here, and in the comment near end of file.